app/models/katello/concerns/hostgroup_extensions.rb

```diff
@@ -76,5 +76,5 @@ def inherited_ancestry_attribute(attribute)
 end
 
 class ::Hostgroup::Jail < Safemode::Jail
-  allow :content_source, :rhsm_organization_label
+  allow :content_source, :rhsm_organization_label, :subscription_manager_configuration_url
```

Comment:
I think there may still be a bug in foreman somewhere as the commit you pointed to @stbenjam should do this i believe.  That said I'm fine with this change.  ACK   @daviddavis any objections?